java/core/src/main/java/co/worklytics/psoxy/impl/SanitizerUtils.java

```diff
@@ -330,7 +330,7 @@ MapFunction getPseudonymizeEmailHeaderToJson(Pseudonymizer pseudonymizer, Transf
             com.google.api.client.util.Preconditions.checkArgument(value instanceof String, "Value must be string");
 
             if (StringUtils.isBlank((String) value)) {
-                return new ArrayList<>();
+                return ""; // empty string
```

Comment:
so this had nothing do do with pseudonym format directly, but maybe bc of poor code structure where we ran urlsafe encoding here (see L350), but in JSON format case we run outside, this when to `"[]"` in that case, where as in URLSAFE case it's left as `[]`???

Reply:
nm, `null` is checked up at L325; so it DOES preserve `null` if header was `null`